modules/dmrpp_module/Chunk.cc

```diff
@@ -209,17 +209,20 @@ size_t chunk_write_data(void *buffer, size_t size, size_t nmemb, void *data) {
  *
  * @note Stolen from the HDF5 library and hacked to fit.
  *
- * @param dest Write the 'inflated' data here
+ * @param destp The pointer to the pointer of the 'inflated' data 
```

Comment:
You could also describe this as a 'value-result' parameter - that is, the parameter is passed in as a value and the code modifies that in the course of execution and then uses the parameter to return the result.